src/ui/Windows/WZPropertySheet.cpp

```diff
@@ -113,6 +114,13 @@ INT_PTR CWZPropertySheet::DoModal()
   return rc;
 }
 
+BOOL CWZPropertySheet::OnInitDialog()
```

Comment:
I'm wondering if we could define a base class for all pwsafe dialogs and perform common init actions there rather than repeat them in every dialog subclass.

Reply:
IIRC, CDialog, propsheets and "wizard" propsheets don't share a useful common ancestor for this.